tests/test-dirs/syntax-document/language-extensions.t/run.t

```diff
@@ -230,18 +233,38 @@ on type t..
   > module SVG = struct end
   > module PNG = struct end
   > let _svg = Hashtbl.add devices "SVG" (module SVG : DEVICE)
-  > let _png = Hashtbl.add devices "PNG" (module PNG : SVG)
+  > let _png = Hashtbl.add devices "PNG" (module PNG : DEVICE)
+  > let sort (type s) (module Set : Set.S with type elt = s) l =
+  >   Set.elements (List.fold_right Set.add l Set.empty)
+  > let make_set (type s) cmp =
+  >   let module S = Set.Make(struct
+  >     type t = s
+  >     let compare = cmp
+  >   end) in
+  >   (module S : Set.S with type elt = s)
   > EOF
 on type picture
   $ syn_doc 1:6 \
   > -filename ./first-class-modules.ml < ./first-class-modules.ml | jq '.value.name'
-  "Record type"
+  "Record Type"
+on module SVG..
+  $ syn_doc 6:6 \
+  > -filename ./first-class-modules.ml < ./first-class-modules.ml | jq '.value'
+  "No documentation found"
 on (module SVG : DEVICE)
-  $ syn_doc 6:43 \
+  $ syn_doc 8:43 \
   > -filename ./first-class-modules.ml < ./first-class-modules.ml | jq '.value.name'
   "First class module"
-on (module PNG : SVG)
-  $ syn_doc 7:43 \
```

Comment:
You could have kept the test for line 6 since it was actually showing incorrect behavior. It should be fixed now.